cmd/gaia/app/app.go

```diff
@@ -216,6 +216,16 @@ func (app *GaiaApp) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.R
 	}
 }
 
+// antehandler for gaia tx validation
+// checks account validity and circuit breaker validity
+func (app *GaiaApp) NewAnteHandler() sdk.AnteHandler {
```

Comment:
Why was this added exactly?

Reply:
Missed to remove it from this PR, will delete this part